src/test/java/ConsolidatorTest.java

```diff
@@ -40,19 +40,19 @@ public void testSimpleCopyCellsToOtherExcel() {
         ArrayList<Excel> excels = new ArrayList<>();
         excels.add(dst);
 
-        new Consolidator(Properties.getMappingFilePath(), src, excels, 0);
+        new Consolidator(Properties.getMappingFilePath(), src, excels);
         dst.save();
 
         XlsxExcelFile excel = new XlsxExcelFile(TestHelper.getWritableFilePath(), 0);
-        assertEquals(src.getCellValue(0,0), excel.getCellValue(0, 0));
-        assertEquals(src.getCellValue(0,1), excel.getCellValue(0, 1));
-        assertEquals(src.getCellValue(0,2), excel.getCellValue(0, 2));
-        assertEquals(src.getCellValue(1,0), excel.getCellValue(1, 0));
-        assertEquals(src.getCellValue(1,1), excel.getCellValue(1, 1));
-        assertEquals(src.getCellValue(1,2), excel.getCellValue(1, 2));
-        assertEquals(src.getCellValue(2,0), excel.getCellValue(2, 0));
-        assertEquals(src.getCellValue(2,1), excel.getCellValue(2, 1));
-        assertEquals(src.getCellValue(2,2), excel.getCellValue(2, 2));
+        assertEquals(src.getCellValue(0,0).toString(), excel.getCellValue(0, 0).toString());
+        assertEquals(src.getCellValue(0,1).toString(), excel.getCellValue(0, 1).toString());
```

Comment:
True, but in the end the program generally will convert string fields containing only numbers to numeric cells, so I'm not sure the test would actually still succeed, were I to add an equals method for CellContent, I suppose I could specify that string cells contraining a numeric string are considered equal to their numeric cell counterpart.